### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ditiselit**
- [DEMO LINK](https://ditiselit.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://ditiselit.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,47 @@
...
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__product">
+        <div class="card__images">
+          <img
+              src="./images/imac.jpeg"
+              alt="images-product"
+          >
```
Comment (volodymyr-soltys97) [inline src/index.html:20]: Add classes for this element
```suggestion
          <img
            class="card__image"
            src="./images/imac.jpeg"
            alt="images-product"
          >
```

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,47 @@
...
+        </div>
+
+        <div class="card__content">
+          <div class="name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:24]: Need to use the semantic tag for text, fix it everywhere

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,147 @@
+// #region page
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:4]: Move font family to the html selector

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,147 @@
...
+
+  font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:6]: Default value, so you can just remove it
```suggestion
```

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,147 @@
...
+
+  --border-color: rgb(243, 243, 243);
+  --blue-color: rgb(0, 172, 220);
+  --dark-blue-color: rgb(6, 11, 53);
+  --grey-color: rgb(97, 96, 112);
+  --white-color: rgb(255, 255, 255);
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:12]: Create the variables with root selector

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,147 @@
...
 }
+
+// #endregion
+
+// #region card
+.card {
+  display: block;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:19]:
```suggestion
```

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,147 @@
...
+  width: 198px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:20]:
```suggestion
  width: 200px;
```

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,147 @@
...
+        margin-right: 4px;
+      }
+
+      &__star:nth-child(5) {
+        margin-right: 0;
+      }
+
+      &--4 .stars__star:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:88]: Move stars styles to the separate file

File: src/styles/index.scss [hunk 9]
```diff
@@ -1,3 +1,147 @@
...
+    line-height: 16px;
+    text-transform: uppercase;
+    color: var(--dark-blue-color);
+  }
+}
+
+// #endregion
+
+// #region button
+button {
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:124]: Need to use the classes selector for button

File: src/styles/index.scss [hunk 10]
```diff
@@ -1,3 +1,147 @@
...
+  width: 166px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:125]:
```suggestion
  width: 100%;
```

File: src/styles/index.scss [hunk 11]
```diff
@@ -1,3 +1,147 @@
...
+  height: 40px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:126]: Use padding instead of hardcode the height
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
This part of the task is incomplete
image
image

File: src/styles/index.scss (hunk 11 above)
Comment (Ditiselit) [reply src/styles/index.scss:104]: I'll leave the code as it is, otherwise it breaks, or show me how to fix it.

**Pull request closed without merging**